dumper/app/src/main/resources/oracle-stats/native/db-objects.sql

```diff
@@ -0,0 +1,63 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  A.con_id "ConId",
+  A.owner "Owner",
+  A.object_type "ObjectType",
+  A.editionable "Editionable",
+  count(1) "Count"
+FROM cdb_objects A
+WHERE A.owner NOT LIKE '%SYS'
+  AND A.object_name NOT LIKE 'BIN$%'
+GROUP BY
+  A.con_id,
+  A.owner,
+  A.object_type,
+  A.editionable
+UNION ALL (
+  SELECT
+    B.con_id "ConId",
+    'SYS' "Owner",
+    'DIRECTORY' "ObjectType",
+    B.editionable "Editionable",
+    count(1) "Count"
+  FROM cdb_objects B
+  WHERE B.owner = 'SYS'
```

Comment:
There are two `owner NOT LIKE '%SYS'` conditions and one `owner = 'SYS'` in this query. Does it mean that the value `EXAMPLE_SYS` would be excluded always or should `B.owner = 'SYS'` be changed to `B.owner LIKE '%SYS'`?

Reply:
We would aim to always exclude `EXAMPLE_SYS`. This is based on: https://github.com/misolt/database-assessment/blob/bqma-dev/scripts/collector/oracle/sql/extracts/dbobjects.sql#L29

The main difference between these is that DMA has a more precise filter, defined inside `exclude_schemas.sql`, while at the time we only use `LIKE '%SYS'`, which excludes most of the system users.